### DISCUSSION TITLE
Meilisearch

### DISCUSSION THREAD

**Discussion by AruneshSingh** [superlinked/VectorHub#93] [VDB Comparison]
Hey everyone, this is the start of discussions related to [Meilisearch](https://www.meilisearch.com/) and their attributes as compared in the table [here](https://vdbs.superlinked.com). 

To add to the discussion please check if there's already a comment with the attribute you want to discuss about, and reply in the thread. Otherwise please create a new comment for the attribute you want to discuss about.

To create an issue, please have a look at the [README](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) for contributing guidelines.

Let's all try to have a fruitful discussion!

**Comment by Kerollmops** [comment]
I have recently fixed and updated a lot of issues and missing stuff about Meilisearch in #240 which were explained in #210.
